docs/cloud/azureai/use-flow-in-azure-ml-pipeline.md

```diff
@@ -127,3 +127,9 @@ ml_client.jobs.create_or_update(pipeline_with_flow)
 :::
 
 ::::
+
+## Difference across flow in prompt flow and pipeline job
+
+In prompt flow, flow runs on runtime (a computing resource usually hosted on a compute instance), which can be regarded exclusive for flow run; while in pipeline job, flow runs on the configured general compute resources (usually compute cluster).
```

Comment:
what do you mean by:   regarded exclusive for flow run?

Reply:
offline synced on this, and updated the wording